packages/pn-personagiuridica-webapp/public/conf/env/config.prod.json

```diff
@@ -0,0 +1,15 @@
+{
+    "API_BASE_URL": "https://webapi.notifichedigitali.it/",
+    "DISABLE_INACTIVITY_HANDLER": false,
+    "MIXPANEL_TOKEN": "005d3e84d0efda94601d38f6d25499cd",
+    "ONE_TRUST_DRAFT_MODE": true,
+    "ONE_TRUST_PP": "9d7b7236-956b-4669-8943-5284fba6a815",
+    "ONE_TRUST_TOS": "112fd95d-6e88-461b-b8fc-534ee4277e96",
+    "OT_DOMAIN_ID": "8d53193a-e4c2-44e5-9fb7-01f08007d083",
+    "PAGOPA_HELP_EMAIL": "notifichedigitali@assistenza.pagopa.it",
+    "SELFCARE_BASE_URL": "https://pnpg.selfcare.pagopa.it",
+    "URL_FE_LOGIN": "https://imprese.notifichedigitali.it/auth/login",
```

Comment:
Should be https://pnpg.selfcare.pagopa.it/auth/login

Reply:
it is in fact correct, since we implemented cdn routing. I proceed to fix the other environment configs